src/lib/glee.ts

```diff
@@ -259,7 +259,7 @@ export default class Glee extends EventEmitter {
   }
 
   private _execErrorMiddleware (emws: ChannelErrorMiddlewareTuple[], index: number, error: Error, message: GleeMessage) {
-    emws.at(index).fn(error, message, (err: Error) => {
+    emws[index as number].fn(error, message, (err: Error) => {
```

Comment:
Watch out for negative indices. `Array.at` supports relative indexing, meaning -1 would not fail but instead it will get the last item on the array: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at#parameters.

Reply:
I don't think there's any way to have a negative `index` with the current code. Should I add a check just in case?

Reply:
Yeah, as you said, not possible with the **current** code. But code continues to evolve so I'd put a check there just in case.